modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -324,7 +324,10 @@ private void handleTxCleanupCommand(TxCleanupCommand cmd, long commandIndex, lon
         Set<RowId> pendingRowIds = txsPendingRowIds.getOrDefault(txId, Collections.emptySet());
 
         if (cmd.commit()) {
-            storage.runConsistently(() -> {
+            storage.runConsistently(locker -> {
+                //TODO Split & sort.
```

Comment:
I'll fix it. It's not really dangerous right now, I guess, because all insertions are protected by external locks in transactions, but nonetheless. Code hygiene is never a bad thing